OTRExporter/OTRExporter/Main.cpp

```diff
@@ -91,8 +91,17 @@ static void ExporterProgramEnd()
 		otrArchive = Ship::Archive::CreateArchive(otrFileName, 40000);
 
 		for (auto item : files) {
+			std::string fName = item.first;
+			if (fName.find("gTitleZeldaShieldLogoMQTex") != std::string::npos && !ZRom(romPath).IsMQ())
```

Comment:
Well I would still have to check for that string in every file of this loop, but I could track whether or not I found it earlier in the loop and not check if I've already found it I suppose. I can also refactor this so it doesn't ever check in a master quest rom (since non-mq roms are the only ones I change this for).